src/dos/dos_files.cpp

```diff
@@ -1422,15 +1422,6 @@ bool DOS_FCBOpen(uint16_t seg, uint16_t offset)
 	if (!DOS_MakeName(shortname, fullname, &drive))
 		return false;
 
-	/* Check, if file is already opened */
```

Comment:
Damn, this is a nice find that we can remove this. Great find and thank you for writing good commit message.